deploy/contents/install/app/install.sh

```diff
@@ -1369,5 +1372,19 @@ if is_service_requested cp-storage-lifecycle-service; then
     echo
 fi
 
+if is_deployment_type_requested aws_eks; then
+
+   if [ $CP_EKS_CSI_DRIVER_TYPE = "efs"] && [ -n $CP_EKS_EFS_CSI_EXECUTION_ROLE]; then
+      print_ok "[Starting install EFS CSI driver in AWS EKS deployment]"
+      create_kube_resource $K8S_SPECS_HOME/efs-csi --ktz
```

Comment:
I would rename `efs-csi` and `fsx-sci` to `cp-system-fs-efs` `cp-system-fs-fsx`, since the main goal of these set of resources is to configure `PV` inside the kube cluster, to then use it as common file system, and not to install driver itself